src/jit/optimizer.cpp

```diff
@@ -3293,7 +3293,7 @@ void Compiler::optUnrollLoops()
  *  not execute a method call.
```

Comment:
Change the comment from  "Return non-zero" to "Return false"
And reword,  "will not" to "could possibly not"